### ISSUE TITLE
For a site hosted inside ASE the Instance Allocation Event is incorrect 

### ISSUE THREAD

**Issue by kaushalp** [ShekharGupta1988/AppLensV2#526]
The instance allocation event timeline seems to be incorrect when the web app is hosted in an ASE. 

![image](https://cloud.githubusercontent.com/assets/4186691/26091053/628e88b0-3a26-11e7-814f-d450173361e0.png)

In this scenario, the site was running on a single instance between 03:35 AM - 9 AM UTC. I think the problem is with the way the data is fetched from the DWAS table. The IIS logs from worker can be used as a reference here. 
Basically in this scenario, the user made a change where they hard-coded the number of instances for the app service plan to 1. They reverted back the change at around 9 AM UTC. 
This can be very misleading when investigating performance related issues.
